metricbeat/module/vsphere/datastore/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"vsphere",
+        "name":"datastore",
+        "rtt":44269
+    },
+    "vsphere":{
+        "datastore":{
+            "example": "datastore"
```

Comment:
Would be nice to have here a full example data set. This can happen in a follow up PR.